packages/create-mg-prompts/src/test/unit/paths.test.ts

```
@@ -0,0 +1,118 @@
+      vi.mocked(homedir).mockReturnValue('/home/test');
+      process.env.XDG_CONFIG_HOME = '/home/test/.config';
+      
+      const expectedPath = '/home/test/.config/claude';
+      setupFileSystem({
+        [expectedPath]: '',
+      });
+
+      const result = await detectClaudeCodePath();
+      expect(result).toBe(expectedPath);
+      
+      delete process.env.XDG_CONFIG_HOME;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)